cubids/workflows.py

```diff
@@ -42,23 +42,29 @@ def validate(
 
     Parameters
     ----------
-    bids_dir
-    output_prefix
-    container
-    sequential
-    sequential_subjects
-    ignore_nifti_headers
+    bids_dir : :obj:`pathlib.Path`
+        Path to the BIDS directory.
+    output_prefix : :obj:`pathlib.Path`
+        Output filename prefix.
+    container : :obj:`str`
+        Container in which to run the workflow.
+    sequential : :obj:`bool`
+        Run the validator sequentially.
+    sequential_subjects : :obj:`list` of :obj:`str`
+        Filter the sequential run to only include the listed subjects.
+    ignore_nifti_headers : :obj:`bool`
+        Ignore NIfTI headers when validating.
     """
     # check status of output_prefix, absolute or relative?
     abs_path_output = True
     if "/" not in str(output_prefix):
         # not an absolute path --> put in code/CuBIDS dir
         abs_path_output = False
         # check if code/CuBIDS dir exists
-        if not Path(str(bids_dir) + "/code/CuBIDS").is_dir():
+        if not (bids_dir / "code" / "CuBIDS").is_dir():
             # if not, create it
-            subprocess.run(["mkdir", str(bids_dir) + "/code"])
-            subprocess.run(["mkdir", str(bids_dir) + "/code/CuBIDS/"])
+            subprocess.run(["mkdir", str(bids_dir / "code")])
+            subprocess.run(["mkdir", str(bids_dir / "code" / "CuBIDS")])
```

Comment:
@mattcieslak do you know why CuBIDS uses `subprocess.run` here instead of `os.makedirs`? Does Datalad not play well with `os.makedirs`?

Reply:
this is odd, I don't remember why it's done this way